zenoh-plugin-ros2dds/src/route_service_cli.rs

```diff
@@ -373,22 +375,22 @@ fn route_dds_request_to_zenoh(
     } else {
         tracing::trace!(
             "{route_id}: routing request {request_id} from DDS to Zenoh (timeout:{query_timeout:#?}) - {} bytes",
-            zenoh_req_buf.len()
+            zenoh_req_buf.to_zslice().len()
```

Comment:
There is no need to call `to_zslice()` to get the length.

Reply:
Missing support from `zenoh`. Fixed in https://github.com/eclipse-zenoh/zenoh/commit/23e36b3fb61b50d9ee90883479858fe8093579ee and I've updated this PR.